src/stores/calculations.ts

```diff
@@ -18,8 +18,8 @@ type CalculationDTO = {
 
 let unsubscribe: Unsubscriber
```

Comment:
Why semicolon at the end of string is not used?

Reply:
Maybe VScode missing auto-format in TS...🤔